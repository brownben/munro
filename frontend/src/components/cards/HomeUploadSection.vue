<template>
  <section class="w-full py-10 bg-white border-t md:py-12 border-main-100">
    <div
      class="flex flex-col items-center justify-between w-10/12 mx-auto md:flex-row-reverse lg:w-3/4 xl:w-2/3"
    >
      <div class="w-full mb-12 text-center md:w-1/2 md:text-left md:mb-0">
        <h2
          class="mb-2 text-4xl tracking-tight text-gray-900 xl:text-5xl font-heading leading-12 xl:leading-14"
        >
          Upload Results in
          <strong class="tracking-normal text-main-600">Seconds</strong>
        </h2>
        <h3 class="mb-2 text-2xl leading-8 text-gray-500 md:mb-3 font-heading">
          No Fuss. No Hassle. Results Created with Ease
        </h3>
        <router-link
          to="/upload/instructions"
          class="mx-0 mt-2 button no-expansion"
          >View Instructions</router-link
        >
      </div>

      <img
        class="w-2/3 md:w-5/12 xl:w-1/3"
        src="@/assets/images/UnDrawUpload.svg"
        alt="Person Bringing Files for Upload"
        loading="lazy"
      />
    </div>
  </section>
</template>
