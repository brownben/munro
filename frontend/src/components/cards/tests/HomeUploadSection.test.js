import { mount } from '@vue/test-utils'
import HomeUploadSection from '@/components/cards/HomeUploadSection.vue'

test('Is a Vue Instance', () => {
  const wrapper = mount(HomeUploadSection, { stubs: ['router-link'] })
  expect(wrapper.isVueInstance()).toBeTruthy()
})

test('Renders Correctly', () => {
  const wrapper = mount(HomeUploadSection, { stubs: ['router-link'] })
  expect(wrapper.element).toMatchSnapshot()
})
