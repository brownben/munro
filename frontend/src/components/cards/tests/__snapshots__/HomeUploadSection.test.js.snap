// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Correctly 1`] = `
<section
  class="w-full py-10 bg-white border-t md:py-12 border-main-100"
>
  <div
    class="flex flex-col items-center justify-between w-10/12 mx-auto md:flex-row-reverse lg:w-3/4 xl:max-w-screen-xl"
  >
    <div
      class="w-full mb-12 text-center md:w-1/2 md:text-left md:mb-0"
    >
      <h2
        class="mb-2 text-4xl tracking-tight text-gray-900 xl:text-5xl font-heading leading-12 xl:leading-14"
      >
        
        Upload Results in
        
        <strong
          class="tracking-normal text-main-600"
        >
          Seconds
        </strong>
      </h2>
       
      <h3
        class="mb-2 text-2xl leading-8 text-gray-500 md:mb-3 font-heading"
      >
        
        No Fuss. No Hassle. Results Created with Ease
      
      </h3>
       
      <router-link-stub
        class="mx-0 mt-2 button no-expansion"
        to="/upload/instructions"
      >
        View Instructions
      </router-link-stub>
    </div>
     
    <img
      alt="Person Bringing Files for Upload"
      class="w-2/3 md:w-5/12 xl:w-1/3"
      loading="lazy"
      src="@/assets/images/UnDrawUpload.svg"
    />
  </div>
</section>
`;
